test/functional/apps/dashboard/_dashboard.js

```diff
@@ -124,4 +123,20 @@ bdd.describe('dashboard tab', function describeIndexTests() {
         });
     });
   });
+
+  bdd.it('add new visualization link', async function checkTitles() {
+    await PageObjects.dashboard.clickAddVisualization();
+    await PageObjects.dashboard.clickAddNewVisualizationLink();
+    await PageObjects.visualize.clickAreaChart();
+    await PageObjects.visualize.clickNewSearch();
+    await PageObjects.visualize.saveVisualization('visualization from add new link');
+
+    const visualizations = PageObjects.dashboard.getTestVisualizations();
+    return PageObjects.common.tryForTime(10000, async function () {
+      const panelTitles = await PageObjects.dashboard.getPanelSizeData();
+      PageObjects.common.log('visualization titles = ' + panelTitles);
```

Comment:
This log of visualization titles just outputs;
```
16:43:01.410: in getPanelSizeData
16:43:02.408: visualization titles = [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
```

If you change it to;
```
      PageObjects.common.log('visualization titles = ' + panelTitles.map(item => item.title));
```
then you get the actual titles including the new 'visualization from add new link'.